.github/workflows/ci.yml

```diff
@@ -58,8 +58,8 @@ jobs:
       if: github.event_name == 'push'
       uses: peter-evans/create-pull-request@v5
       with:
-        title: 'Regenerate mocks'
-        commit-message: 'Regenerate mocks'
+        title: 'chore: regenerate mocks'
```

Comment:
I think this is more a matter of style than anything, but I like better `chore: Regenerate mocks.`, removing the CC prefix, it still looks like a sentence.

Reply:
the [spec](https://www.conventionalcommits.org/en/v1.0.0/), dependabot, and renovate don't capitalize, though ;-)

- https://github.com/canonical/ubuntu-desktop-installer/pull/2017
- https://github.com/canonical/ubuntu-desktop-installer/pull/2021

Reply:
I'm the weirdo then :D

Reply:
Nah, I recall Daniel made the same comment at the engineering sprint. :) I wish GitHub had some nice label-like rendering for conventional commit prefixes...